contact.html

```diff
@@ -0,0 +1,180 @@
+      integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>contact</title>
+  </head>
+
+  <body>
+    <header class="container justify-content-between py-4">
+      <nav class="navbar navbar-light navbar-expand-md">
+        <div class="container">
+          <a
+            href="index.html"
+            class="navbar-brand d-flex flex-column flex-md-row align-items-center color"
+          >
+            <h2 class="bold-text">
+              <span class="bg-primary rect d-inline-block me-2"></span>Maya
+              Nelson
+            </h2>
+            <span class="d-none d-md-block px-1">/</span>
+            <span class="text-uppercase fs-6">project manager</span>
+          </a>
+          <button
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+            class="navbar-toggler"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <!-- navbar for bigger screen -->
+          <div
+            id="navbarSupportedContent"
+            class="collapse navbar-collapse justify-content-end"
+          >
+            <ul class="navbar-nav text-uppercase">
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">About me</a>
+              </li>
+              <li class="nav-item">
+                <a href="resume.html" class="nav-link">Resume</a>
+              </li>
+              <li class="nav-item">
+                <a href="projects.html" class="nav-link">Projects</a>
+              </li>
+              <li class="nav-item">
+                <a href="contact.html" class="nav-link active">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main class="container-fluid py-3 mx-auto background-main">
+      <h1 class="text-center py-5">
+        <span class="bg-primary rect d-inline-block me-2"></span>Let's talk
+      </h1>
+
+      <form action="" class="container form-container">
+        <div class="row bg-white p-3 p-md-5">
+          <div class="col-12 col-md-6 mb-4">
+            <label for="first-name"
+              >First Name <span class="text-primary">*</span></label
+            >
+            <input type="text" class="form-control" id="first-name" required />
```

Comment:
Your form looks great but again, write your own CSS to follow the design exactly. When the input is active, it should have a bottom blue line, not a black box. 
You'll need the pseudo-element `:focus` to style this.